src/components/common/MaintenanceBar.vue

```diff
@@ -1,23 +1,36 @@
 <template>
-  <div v-if="session.maintenanceBar && show" class="maintenance-bar">
-    <i></i>
-    <p>
-      We've identified problems with our servers that are causing issues for
-      some of our users. We apologize for the disruption and are working on a
-      fix.
-    </p>
-    <a class="close">
-      <i class="material-icons" @click="close">close</i>
-    </a>
+  <div>
+    <div v-if="session.maintenanceBar && show" class="maintenance-bar">
+      <i></i>
+      <p>
+        We've identified problems with our servers that are causing issues for
+        some of our users. We apologize for the disruption and are working on a
+        fix.
+      </p>
+      <a class="close">
+        <i class="material-icons" @click="close">close</i>
+      </a>
+    </div>
+    <div v-if="maintenance.length > 0 && show" class="maintenance-bar" v-bind:class="maintenance[0].type">
```

Comment:
Why did you limit this just one record? Would it be possible to show multiple messages?

Reply:
> Why did you limit this just one record? Would it be possible to show multiple messages?

Sure, I'll check the needed css so we can stack multiple messages.